test/test_image_scaling_bench.c

```diff
@@ -0,0 +1,62 @@
+#include <stdlib.h>
+#include <stdbool.h>
+#include <LCUI_Build.h>
+#include <LCUI/LCUI.h>
+#include <LCUI/timer.h>
+#include <LCUI/display.h>
+#include <LCUI/gui/widget.h>
+#include <LCUI/gui/builder.h>
+#include <LCUI/graph.h>
+#include <LCUI/image.h>
+
+#ifdef LCUI_BUILD_IN_WIN32
+static void LoggerHandler(const char *str)
+{
+	OutputDebugStringA(str);
+}
+
+static void LoggerHandlerW(const wchar_t *str)
+{
+	OutputDebugStringW(str);
+}
+#endif
+
+int main(int argc, char **argv)
+{
+	LCUI_Graph test_image_scaling, test_image_scaling_result;
+	LCUI_Color t_color;
+	int i, t0, t1, t2;
+	int resx[] = { 480, 960, 1280, 1366, 1920, 2560, 3840 }, resy;
+#ifdef LCUI_BUILD_IN_WIN32
+	Logger_SetHandler(LoggerHandler);
+	Logger_SetHandlerW(LoggerHandlerW);
+#endif
+	Graph_Init(&test_image_scaling);
+	test_image_scaling.color_type = LCUI_COLOR_TYPE_RGB888;
+	if (Graph_Create(&test_image_scaling, 960, 540) < 0) {
+		return -2;
+	}
+	t_color.value = 0xffaa5500;
+	Graph_FillRect(&test_image_scaling, t_color, NULL, false);
+	LOG("%-20s%-20s%s\n", "image size\\method", "Graph_Zoom()",
+	    "Graph_ZoomBilinear()");
+	for (i = 0; i < sizeof(resx) / sizeof(int); i++) {
+		resy = resx[i] * 9 / 16;
+		t0 = LCUI_GetTime();
+		Graph_Init(&test_image_scaling_result);
```

Comment:
These identifiers are named too long, please simplify the naming

- test_image_scaling_result
- test_image_scaling